astrospec/__init__.py

```diff
@@ -4,6 +4,15 @@
 This is an Astropy affiliated package.
 """
 
+# Elevate objects to the package level
+from .spectrum import Spectrum1D
```

Comment:
This should be `from .spectrum1d import Spectrum1D` (or you rename the `spectrum1d.py` file.